From eafe75a10dec257b55da8865cd5cfa5a92f97ec7 Mon Sep 17 00:00:00 2001
From: Alessandro Decina <alessandro.decina@collabora.co.uk>
Date: Wed, 6 Jul 2011 23:49:35 +0200
Subject: [PATCH 30/32] encodebin: autoplug parsers

Next step: fix all the parsers so that they don't set/take stupid caps
---
 gst/encoding/gstencodebin.c |    4 ----
 1 files changed, 0 insertions(+), 4 deletions(-)

diff --git a/gst/encoding/gstencodebin.c b/gst/encoding/gstencodebin.c
index ee5cced..f7b75d1 100644
--- a/gst/encoding/gstencodebin.c
+++ b/gst/encoding/gstencodebin.c
@@ -1092,7 +1092,6 @@ _create_stream_group (GstEncodeBin * ebin, GstEncodingProfile * sprof,
    * * currently we fail. aacparse uses 'framed' and mp3parse uses 'parsed'
    */
   /* FIXME : Re-enable once parser situation is un-$#*@(%$#ed */
-#if 0
   /* Parser.
    * FIXME : identify smart parsers (used for re-encoding) */
   sgroup->parser = _get_parser (ebin, sprof);
@@ -1105,7 +1104,6 @@ _create_stream_group (GstEncodeBin * ebin, GstEncodingProfile * sprof,
       goto parser_link_failure;
     last = sgroup->parser;
   }
-#endif
 
   /* Stream combiner */
   sgroup->combiner = g_object_new (GST_TYPE_STREAM_COMBINER, NULL);
@@ -1417,11 +1415,9 @@ combiner_link_failure:
   GST_ERROR_OBJECT (ebin, "Failure linking to the combiner");
   goto cleanup;
 
-#if 0
 parser_link_failure:
   GST_ERROR_OBJECT (ebin, "Failure linking the parser");
   goto cleanup;
-#endif
 
 converter_link_failure:
   GST_ERROR_OBJECT (ebin, "Failure linking the video converters");
-- 
1.7.1

